docs/source/guide/s3-example-bucket-policies.rst

```diff
@@ -21,9 +21,9 @@ policies are defined using the same JSON format as a resource-based IAM policy.
 Retrieve a bucket policy
 ========================
 
-Retrieve a bucket's policy by calling the AWS SDK for Python 
-``get_bucket_policy`` method. The method accepts a parameter that specifies 
-the bucket name.
+Retrieve a bucket's policy by calling the AWS SDK for Python
+:py:meth:`~S3.Client.get_bucket_policy` method. The method accepts a parameter
```

Comment:
Do these render correctly with our version of Sphinx?

Reply:
They work for me, so I sure hope so. See http://sheppycloud.aka.corp.amazon.com:4006/guide/s3-example-bucket-policies.html#set-a-bucket-policy for this spot on my test server.